### ISSUE TITLE
Closing the listener

### ISSUE THREAD

**Issue by quasiben** [rapidsai/ucx-py#255]
I am trying to build [benchmarks](https://github.com/rapidsai/ucx-py/pull/254)/[tests](https://github.com/rapidsai/ucx-py/pull/233) with a well defined `server` and `client` running in separate processes.  The `server` needs to create a listener and respond appropriately.  The `client`, creates an endpoint pointing to the listener ip/port.

**Comment by quasiben** [comment]
hit the green button before I was finished, the comment is now updated

**Comment by pentschev** [comment]
I'm not sure as to the status of this, is it still relevant @quasiben ?

**Comment by madsbk** [comment]
> I'm not sure as to the status of this, is it still relevant @quasiben ?

Yes I think it is an useful feature, I just haven't had time to work on it.
